theme/parts/tabbox.css

```diff
@@ -0,0 +1,8 @@
+/*Tabbox*/
+
+@namespace xul "http://www.mozilla.org/keymaster/gatekeeper/there.is.only.xul";
+
+/* Main tab content */
+.tabbrowser-tabbox {
```

Comment:
I would use the card's radius for this, something like this might work:
```css
#tabbrowser-tabbox {
	@media -moz-pref("sidebar.revamp.round-content-area") {
		--border-radius-medium: var(--gnome-card-radius) !important;	
	}
}
```